ApiTests/Utils/ApiUtils.cs

```diff
@@ -9,7 +9,10 @@ internal class ApiUtils
         public static RestResponse SendGetRequest(string resource)
         {
             //implement a simple get request
-            return null;
+            return client.Execute(new RestRequest(resource, Method.Get)
+            { RequestFormat = DataFormat.Json });
```

Comment:
can be removed, as we are not passing anything in this request
also get request doesn't really work with json, it carries data in the single URL line